vsts/build-release-artifacts.yml

```diff
@@ -33,70 +33,258 @@ phases:
-            if ($found -eq $false) { throw "File not found '$Cert'" }
-            Write-Host "Importing cert from '$Cert' into store '$Store'"
-        
-            Import-PfxCertificate `
-                -FilePath $Cert `
-                -CertStoreLocation $Store `
-                -Password $Passphrase
-        }
-        
-        $certpwd = $env:certpwd
-        
-        if ([System.String]::IsNullOrWhiteSpace($certpwd) -eq $true) {
-            throw "Empty passphrase. Make sure it's hooked up."
-        }
-        
-        $passphrase = ConvertTo-SecureString -String $certpwd -AsPlainText -Force
-        $store = 'Cert:\LocalMachine\My'
-        $location = $env:agent_tempdirectory
-        $authcert = Join-Path $location 'iothubsdksign-auth-base64.pfx'
-        $reqcert = Join-Path $location 'iothubsdksign-req-base64.pfx'
-        
-        Import-Certificate `
-            -Cert $authcert `
-            -Passphrase $passphrase `
-            -Store $store
-        
-        Import-Certificate `
-            -Cert $reqcert `
-            -Passphrase $passphrase `
-            -Store $store
-
+  - task: PowerShell@2
+    inputs:
+      filePath: '$(Build.StagingDirectory)/scripts/csharp/vsts/install-signing-certificates.ps1'
     displayName: 'Install Signing Certificates'
     env:
         certpwd: $(certpassphrase)
 
   - powershell: |
-      . $(Build.StagingDirectory)/build-tools/csharp/new/SignTools.ps1
+      . $(Build.StagingDirectory)/scripts/csharp/new/SignTools.ps1
       md $env:ESRP_LOG_DIR
       md $env:AZURE_IOT_LOCALPACKAGES
       Set-Location $(Build.SourcesDirectory)
       $(Build.SourcesDirectory)/build.ps1 -configuration Release -sign -verbosity d
 
     displayName: 'Build, Package, Sign'
     env:
-        ESRP_AUTH_CONFIG: $(Build.StagingDirectory)/build-tools/csharp/new/json/Auth.json
-        ESRP_POLICY_CONFIG: $(Build.StagingDirectory)/build-tools/csharp/new/json/Policy.json
-        ESRP_INPUT_JSON_DIR: $(Build.StagingDirectory)/build-tools/csharp/new/json
-        ESRP_LOG_DIR: $(Build.StagingDirectory)/build-tools/csharp/new/logs
-        AZURE_IOT_LOCALPACKAGES: $(Build.StagingDirectory)/build-tools/csharp/nuget_local
+        ESRP_AUTH_CONFIG: $(Build.StagingDirectory)/scripts/csharp/new/json/Auth.json
+        ESRP_POLICY_CONFIG: $(Build.StagingDirectory)/scripts/csharp/new/json/Policy.json
+        ESRP_INPUT_JSON_DIR: $(Build.StagingDirectory)/scripts/csharp/new/json
+        ESRP_LOG_DIR: $(Build.StagingDirectory)/scripts/csharp/new/logs
+        AZURE_IOT_LOCALPACKAGES: $(Build.StagingDirectory)/scripts/csharp/nuget_local
 
   - task: PublishBuildArtifacts@1
     displayName: 'Publish Signed Nuget Packages'
     inputs:
         PathtoPublish: '$(Build.SourcesDirectory)/bin/pkg'
         ArtifactName: 'nuget'
-        publishLocation: 'Container'
\ No newline at end of file
+        publishLocation: 'Container'
+
+### Linux build ###
+- phase: LINUX
```

Comment:
Can we avoid this duplication and have a yaml that triggers another pipeline with parameters?

If that's not possible, please merge all yamls into a single one that will have parameters that will allow release. This runs the risk of getting out of sync with the nightly and CI build systems.